### DISCUSSION TITLE
如何保養自己的眼睛？

### DISCUSSION THREAD

**Discussion by aa103967622** [1101-GS4719-C/Python-Community-GS4719-C#1429] [General]
從小家長會提醒我們，保養眼睛相當重要，這裡跟大家分享一個隨時隨地都能用的方法。眼保健操，旨在改善眼部肌肉的緊張和血液循環，提供眼睛的保健。眼睛經常長時間注視電腦或智能手機螢幕可能導致眼睛疲勞和不適，這些眼球運動操可以在工作中或自宅放鬆時進行，相對容易且安全。

這些眼球運動操包括一閃一閃眨眼操、遠近目視訓練法、大拇指階段性滑動和轉轉8循環操。這些動作旨在放鬆眼部肌肉、改善血液循環、調節眼睛的對焦能力，以減輕眼睛疲勞和促進眼部健康。

參考來源：https://heho.com.tw/archives/129338

**Comment by TW-Maverick** [comment]
感謝分享，我覺得上了大學後，眼睛有越來越疲勞的感覺，應該是很常時間坐在筆電前面導致，現在的作業、程式或專題都需要靠電腦完成，不像以前國高中是書本的形式。
我還記得在國小時，會有特定的時段，大家站在教室外，看向遠方，以前都覺得是浪費時間，現在覺得那時我真蠢！

**Comment by alicia0725** [comment]
大學之後常常睡前發現眼睛很酸痛一直流眼淚等等，不知道是不是因為每天都在看電腦螢幕跟手機螢幕，我自己的方法是如果整天都戴隱形眼鏡看螢幕的話，我會在睡覺前帶蒸汽眼罩，讓眼睛足夠的放鬆，推薦大家試試看！

**Comment by Alexgns** [comment]
長時間注視電腦、手機或其他數碼螢幕容易引起眼睛疲勞和散光。每隔20分鐘，注視遠處物體約20秒，讓眼睛休息一下，記得保持正常的眨眼頻率，以保持眼睛表面的濕潤。

**Comment by alstinlin** [comment]
這是網路上找到保養眼睛的方法，我覺得還蠻不錯的1. 使用電腦時，眼睛應盡量跟螢幕平視，2. 每工作一小時，可以休息10～15分鐘，避免在光線微弱或黑暗處使用手機。3. 避免長時間在昏暗的地方看手機螢幕，因為會造成黃斑部損傷，最後如果像我平時常使用手機或平板，也可加強防護藍光，選戴抗藍的光眼鏡

**Comment by ALAN30359704** [comment]
其實我覺得最好保養眼睛的方式就是減少使用電子產品對眼睛造成的傷害，因為我們高頻率的使用，所以影響應該更大有效地控制使用時間我認為是解決的根本辦法。

**Comment by Tzuhsuannnnn** [comment]
上大學之後越來越多的功課要透過平板或電腦來完成，增加了對眼睛的負擔。我每天都有在吃葉黃素，雖然還沒有很有感，但應該還是不無小補，大家可以去藥局或康是美之類，有許多不同廠商出的葉黃素可以選擇。

**Comment by Miinteaa** [comment]
以前高三備考時因為長時間用眼再加上睡眠不足，很容易有眼睛不舒服的情況、甚至引發頭痛等其他問題，後來查詢資料過後發現是用眼過度導致眼部肌肉過於疲勞。
若是想要改善眼部不舒服的情況，除了適當休息、避免熬夜、保持良好用眼習慣以外，也可以上網找找穴道按摩的方式，其實蠻有效的！

**Comment by brian431** [comment]
眼睛的保養確實十分重要，尤其現在這個時代幾乎所有工作都需要大量用眼。這讓平常的保護保養及休息變得更加重要。希望大家都能養成健康的用眼習慣。

**Comment by brian431** [comment]
眼睛的保養確實十分重要，尤其現在這個時代幾乎所有工作都需要大量用眼。這讓平常的保護保養及休息變得更加重要。希望大家都能養成健康的用眼習慣。

**Comment by Aaron0810** [comment]
我家裡有一張板子就是用來那個讓眼睛轉轉轉的那個，有個像是10芒星之類的東西然後有順序1~10之類的要照順序看，小時候睡前都會看個好幾圈才睡，雖然我本人是有近視的說服力可能沒那麼高，但感覺那個應該是真的有用。

**Comment by youtinglin** [comment]
不得不說現代人盯著螢幕的時間變長了，有時候上課、做報告都需要看螢幕，真的蠻傷眼睛的，除了傳統的一些保健知識，可以藉由食物的攝取，包含葉黃素(南瓜、菠菜):吸收強光的能量，降低對眼睛的傷害。花青素(蔓越莓、葡萄):提高視網膜對光線刺激的適應力，舒緩眼睛疲勞。牛磺酸(枸杞子、貝類):維持視網膜功能正常，強化角膜的自我修復。蝦紅素(鮭魚、蝦):輔助葉黃素保護黃斑部，增進對焦調節力。omega-3(魚油、堅果):保護視網膜感光細胞，確保黃斑部健康。
